setup.py

```diff
@@ -10,7 +10,12 @@
 with open(join(dirname(__file__), "README")) as f:
     long_description = f.read().strip()
 
-install_requires = ["urllib3>=1.21.1", "certifi"]
+install_requires = [
+    "urllib3>=1.21.1",
+    "certifi",
+    # Async is supported on Python 3.6+
+    "aiohttp; python_version>='3.6'",
```

Comment:
I'd suggest adding a version range for aiohttp like ``aiohttp<4.0,>=3.5.4`` to avoid getting a potentially unknown package and having an API break occur.

Reply:
Good idea, I'm planning on adding an extra for async via `elasticsearch[async]` so will fix then!